cmd/nerdctl/compose_down_linux_test.go

```diff
@@ -24,6 +24,44 @@ import (
 	"github.com/containerd/nerdctl/pkg/testutil"
 )
 
+func TestComposeDownRemoveUsedNetwork(t *testing.T) {
+	base := testutil.NewBase(t)
+
+	var (
+		dockerComposeYAMLOrphan = fmt.Sprintf(`
+version: '3.1'
+
+services:
+  test:
+    image: %s
+    command: "sleep infinity"
+`, testutil.AlpineImage)
+
+		dockerComposeYAMLFull = fmt.Sprintf(`
+%s
+  orphan:
+    image: %s
+    command: "sleep infinity"
+`, dockerComposeYAMLOrphan, testutil.AlpineImage)
+	)
+
+	compOrphan := testutil.NewComposeDir(t, dockerComposeYAMLOrphan)
+	defer compOrphan.CleanUp()
+	compFull := testutil.NewComposeDir(t, dockerComposeYAMLFull)
+	defer compFull.CleanUp()
+
+	projectName := fmt.Sprintf("nerdctl-compose-test-%d", time.Now().Unix())
+	t.Logf("projectName=%q", projectName)
+
+	// orphanContainer := fmt.Sprintf("%s_orphan_1", projectName)
+
+	base.ComposeCmd("-p", projectName, "-f", compFull.YAMLFullPath(), "up", "-d").AssertOK()
+	// defer base.ComposeCmd("-p", projectName, "-f", compFull.YAMLFullPath(), "down", "-v").Run()
```

Comment:
should be deleted ?

Reply:
yep，I pushed this commit just want to try to see if I can pass the CI test~ 

Will refactor this